compiler/src/dotty/tools/dotc/core/tasty/TreeUnpickler.scala

```diff
@@ -1021,7 +1021,12 @@ class TreeUnpickler(reader: TastyReader,
               val expr = ifBefore(end)(readTerm(), EmptyTree)
               Return(expr, Ident(from.termRef))
             case TRY =>
```

Comment:
Maybe, also add a test with no `finally` to exercise this path. I found tests for `try ... catch` but not within `inline def`

Reply:
Added